workers/limiter/tests/index.test.ts

```diff
@@ -118,6 +115,7 @@ describe('Limiter worker', () => {
       mockedEvents.push(createEventMock());
     }
     await eventsCollection.insertMany(mockedEvents);
+    await repetitionsCollection.insertMany(mockedEvents);
```

Comment:
its unclear that this method will add N events + N repetitions. Lets add a separate param `repetitionsToMock`